metrics-operator/api/v1/keptnmetricsprovider_types.go

```diff
@@ -26,8 +26,8 @@ import (
 // KeptnMetricsProviderSpec defines the desired state of KeptnMetricsProvider
 type KeptnMetricsProviderSpec struct {
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:validation:Pattern:=prometheus|thanos|dynatrace|datadog|dql
-	// Type represents the provider type. This can be one of prometheus, thanos, dynatrace, datadog, dql.
+	// +kubebuilder:validation:Pattern:=prometheus|thanos|cortex|dynatrace|datadog|dql
```

Comment:
this may be my ocd, but since this list is destined to grow and it will be autogenerated in all docs from here, should we think of ordering the providers alphabetically for an easy read trough?

Reply:
i guess we can...

Reply:
@RealAnna adapted